docs/source/concept_models/vivarium_swissre_breastcancer/concept_model.rst

```diff
@@ -394,6 +396,9 @@ Using OR value of 1.89 and P as 0.3
     - I'm wondering if the upper and lower limits of the truncated normal distributions should be narrower? What we are modelling here are the 'guideline times' to next screening, hence shouldn't they fall within the bounds of 1 year or 2 years according to the screening tree? Currently for someone who is in branch A, B, or C and supposed to have yearly screens, their next scheduled screen can be as far in the future as two years (with an upper bound of 700 days). 
 
     - I'm wondering if the Marketscan data, where we got the empirical distributions from, is giving us the time interval between screens that the patient actually showed up to? (which in our model is a combintation of time to next scheduled screening + probabiltiy of showing up)
+    
+    - For individuals survive more than 10 years after the diagnosis and treatment, 
+    send them to susceptible state or we could create a remission state and assume they will never have a chance to be screened during the simulation.
```

Comment:
They are going into a remission state and then continuing to get screened in the model (though the result of the screening will always say in remission).

Reply:
This keeps things simple, and also lets us count how much screening is happening.